src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt

```diff
@@ -1067,35 +1143,21 @@ class CourtSentencingService(
           ),
         )
 
-        response.takeIf { it.courtEventIds.isNotEmpty() }?.also {
-          val mapping = CourtAppearanceRecallMappingsDto(
-            nomisCourtAppearanceIds = it.courtEventIds,
-            dpsRecallId = recallId,
-          )
-          createMapping(
-            mapping = mapping,
-            telemetryClient = telemetryClient,
-            retryQueueService = courtSentencingRetryQueueService,
-            eventTelemetry = telemetryMap,
-            name = EntityType.COURT_APPEARANCE_RECALL.displayName,
-            postMapping = courtCaseMappingService::createAppearanceRecallMappings,
-            log = log,
-          )
-        }
-
-        queueService.sendMessageTrackOnFailure(
-          queueId = "fromnomiscourtsentencing",
-          eventType = "courtsentencing.resync.sentence-adjustments",
-          message = SyncSentenceAdjustment(
+        tryCreateAppearanceMappingsAndNotifySentenceAdjustmentsAndClonedCases(
+          mappingsWrapper = RecallAppearanceAndCreateMappingsWrapper(
+            sentenceAdjustmentsActivated = response.sentenceAdjustmentsActivated,
+            mappings = CourtAppearanceRecallMappingsDto(
+              nomisCourtAppearanceIds = response.courtEventIds,
+              dpsRecallId = recallId,
+            ),
             offenderNo = offenderNo,
-            sentences = response.sentenceAdjustmentsActivated.map {
-              SentenceIdAndAdjustmentIds(
-                sentenceId = it.sentenceId,
-                adjustmentIds = it.adjustmentIds,
-              )
-            },
+            // TODO
```

Comment:
once we set this we need a whole set of tests showing what happens when the case is cloned due to it being on a old booking